dm/checker/checker.go

```diff
@@ -131,6 +136,14 @@ func (c *Checker) Init(ctx context.Context) (err error) {
 			return terror.ErrTaskCheckGenTableRouter.Delegate(err)
 		}
 
+		if instance.cfg.OnlineDDL && c.onlineDDL == nil {
+			c.onlineDDL, err = onlineddl.NewRealOnlinePlugin(c.tctx, instance.cfg)
```

Comment:
c.onlineDDL.storage may be partially inited, db in `c.onlineDDL.storage` should be closed

Reply:
In fact, we have not used the **db** in `onlineDDL.storage`.

Reply:
previous bug, maybe fix it in this pr.

Reply:
#4142

Reply:
fixed in https://github.com/pingcap/tiflow/pull/4065/commits/6b6ad535ab1ee2e67d0f6001a452c31d347c05b4